### PR TITLE
Add a link to report an Issue

### PR DESCRIPTION
**Pull request by hi-rustin**
close https://github.com/rust-lang/crates.io/issues/3129

### CODE REVIEW

**Comment by hi-rustin** [comment]
image

I'm not sure we should force a line break?

File: app/components/footer.hbs [hunk 1]
```diff
@@ -8,10 +8,12 @@
     <span local-class="sep">|</span>
     <a href='https://www.rust-lang.org/policies/security'>Report a security issue</a>
     <span local-class="sep">|</span>
+    <a href='https://github.com/rust-lang/crates.io/issues/new/choose'>Report an issue</a>
```
Comment (JohnTitor) [inline app/components/footer.hbs:11]: I couldn't attend the meeting so I have no context, but I think it's clearer to say "Report a general issue", considering we have "Report a security issue". And I'd prefer to place the security link next to the general link.

Comment (hi-rustin) [reply app/components/footer.hbs]: Make sense. Addressed. Thanks for your review!

**Review by JohnTitor — APPROVED** [review]
The change itself looks good, though I'm going to defer the final review to another reviewer as I'm not sure if we need a style change.

**Comment by hi-rustin** [comment]
PC:
image

mobile:
image

**Comment by hi-rustin** [comment]
r? @jtgeibel

Could you please take a look? Thanks!

**Comment by hi-rustin** [comment]
@Turbo87 

Could you please take a look? Thanks!

**Review by Turbo87 — APPROVED** [review]
probably not the final state of the footer, but I guess this is a good intermediate step :)

**Comment by epage** [comment]
As a user, "report an issue" sounds definitive, that I know there is an issue.  I don't remember what issue led to #3129 but I had written "I was trying to get support with a potential problem with crates.io".  The new link would not have helped me in this case.  I probably would still click "send an email" because (1) it sounds closest to a means of support and (2) there is no clarification on when emailing should be used.

**Comment by hi-rustin** [comment]
> As a user, "report an issue" sounds definitive, that I know there is an issue. I don't remember what issue led to #3129 but I had written "I was trying to get support with a potential problem with crates.io". The new link would not have helped me in this case. I probably would still click "send an email" because (1) it sounds closest to a means of support and (2) there is no clarification on when emailing should be used.

The link I added here is `report a general issue
`, maybe we do need a link that explains how to use them. This is not the final footer state, maybe we can consider it in a future design?

Also, I added it just thinking we could jump to here via a direct link. It's also what you mentioned in the possible alternative solutions.

**Pull request merged**